src/platform/i386/boot_comp.c

```diff
@@ -144,7 +147,7 @@ kern_boot_comp(void)
 
 	printk("\tCapability table and page-table created.\n");
 
-	ret = boot_pgtbl_mappings_add(ct, pgtbl, BOOT_CAPTBL_BOOTVM_PTE, "booter VM", mem_bootc_start(),
+	ret = boot_pgtbl_mappings_add(ct, BOOT_CAPTBL_SELF_PT, BOOT_CAPTBL_BOOTVM_PTE, "booter VM", mem_bootc_start(),
```

Comment:
I don't think that we want the boot component to be in global memory.  I think that we want it to have access to global memory through a captbl and pgtbl.  The `_SELF_PT` cap is the offset of the boot component's page-table.  Not of only a "container" of resources.

So we want to keep the self pt the same as it was as we want it in local memory, but we want the boot component to have access to the component structure through references to separate (i.e. not the component's pgtbl) references  to a pgtbl and captbl.